lib/domodule.js

```diff
@@ -169,4 +169,20 @@ export default class Domodule { // eslint-disable-line no-unused-vars
       }
     }
   }
+
+  hide(el = this.el) {
+    if (!(el instanceof Node)) {
+      el = this.find(el);
+    }
+
+    el.style.display = 'none';
```

Comment:
Can we save the current display style on the element and use that value in show so we don't have to pass it in?